ChatBoatSwiftAI/ChatBoatSwiftAI/ViewController.swift

```diff
@@ -53,29 +81,44 @@ extension ViewController: UITableViewDataSource {
     }
 
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        
+        guard let animationCell = tableView.dequeueReusableCell(withIdentifier: "DotAnimationCell", for: indexPath) as? DotAnimationCell else
```

Comment:
Create cell properly when ever it comes in use. not on top.
Currently it is on top it will create every time while it is not in use.